content/en/docs/getting-started/_index.md

```diff
@@ -74,25 +82,40 @@ sudo systemctl enable microshift --now
 {{< /tabs >}}
 <br/>
 
-To access the cluster, install the OpenShift client or kubectl if you haven't done so yet:
+### Install Clients
+
+To access the cluster, install the OpenShift client or kubectl if you haven't done so yet.
 
 ```Bash
 curl -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz
 tar -xzvf oc.tar.gz
 sudo install -t /usr/local/bin {kubectl,oc}
 ```
 
-Depending on the user that will be administrating the system it may be required to copy the kubeconfig to a location that can be accessed by the user.
+### Copy or Set Kubeconfig
+
+For regular users, copy the kubeconfig to a location that can be accessed without administrator privilege.
```

Comment:
These lines below, including the special casing, could be simplified to
```
sudo install -v -D -b -o $(id -u) -g $(id -g) /var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config
```

Reply:
Doesn't work 100%. See earlier comment.

Reply:
ok, i kept with the ~/.kube/config & chown